app/controllers/sealions.rb

```diff
@@ -0,0 +1,17 @@
+require 'sinatra'
+
+get '/sealions' do
```

Comment:
Under typical naming conventions, these would be
get '/sealions' # get all sea lions
get '/sealions' # get some sea lions
post '/sealions' # create a sea lion
put '/sealions/:id' # update a sea lion
delete '/sealions/:id' # :(